bliss/surveys/dc2.py

```diff
@@ -293,6 +186,160 @@ def test_dataloader(self):
             self.test_dataset, batch_size=self.batch_size, num_workers=self.num_workers // 2
         )
 
+    def get_plotting_sample(self, image_idx):
+        if self.image_files is None or self.bg_files is None:
+            self._load_image_and_bg_files_list()
+
+        (
+            image,
+            bg,
+            tile_dict,
+            full_cat,
+            wcs,
+            _,
+            match_id,
+            psf_params,
+        ) = load_image_and_catalog(
+            image_idx,
+            self.image_files,
+            self.bg_files,
+            self.image_lim,
+            self.bands,
+            self.n_bands,
+            self.cat_path,
+            self.min_flux_threshold,
+        )
```

Comment:
So many arguments/return values/lines. Can this be refactored somehow?

Reply:
return a dict to address this